pandas/core/ops/array_ops.py

```diff
@@ -136,6 +136,8 @@ def na_arithmetic_op(left, right, op, str_rep: str):
     left : np.ndarray
     right : np.ndarray or scalar
     str_rep : str or None
+    is_cmp : bool, default False
```

Comment:
on your list is to remove all of the str_rep passing around right?

Reply:
Yes.  ive got a branch that does that, but it isnt passing yet